kube-web.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: caption-generator-web-deployment
+spec:
+  selector:
+    matchLabels:
+      app: caption-generator-web
+  replicas: 1
+  template: 
+    metadata:
+      labels:
+        app: caption-generator-web
+    spec:
+      containers:
+      - name: caption-generator-web
+        image: nheidloff/caption-generator-web:latest
```

Comment:
we are publishing all images to the docker hub MAX organization. I will add the webapp to the org as well.